### PR TITLE
Canbus: add a new vehicle neolix

### PR DESCRIPTION
**Pull request by wangjia1990**

### CODE REVIEW

**Comment by lx18233184051** [comment]
lint: modules/canbus/vehicle/neolix_edu/protocol/aeb_frontwheelspeed_353_test.cc:48:  Lines should be <= 80 characters long

File: modules/canbus/vehicle/neolix_edu/neolix_edu_controller.cc [hunk 1]
```diff
@@ -0,0 +1,590 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
```
Comment (macDure) [inline modules/canbus/vehicle/neolix_edu/neolix_edu_controller.cc:2]: 2020，please change all files.

File: modules/canbus/vehicle/neolix_edu/protocol/BUILD [hunk 2]
```diff
@@ -0,0 +1,232 @@
+load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "canbus_neolix_edu_protocol",
+    srcs = glob(
+        ["*.cc"],
```
Comment (macDure) [inline modules/canbus/vehicle/neolix_edu/protocol/BUILD:9]: please list all .cc files in srcs and hdrs

**Review by macDure — CHANGES_REQUESTED** [review]

**Review by macDure — APPROVED** [review]

**Comment by Capri2014** [comment]
@wangjia1990 @macDure I am merging this one in, but please have the correct commit message format requirement for next pr, it should be: `ModuleName: Commit messages.`

**Pull request merged**